platform/darwin/src/MGLCircleStyleLayer.h

```diff
@@ -15,20 +16,41 @@ typedef NS_ENUM(NSUInteger, MGLCircleStyleLayerCirclePitchScale) {
 
 @interface MGLCircleStyleLayer : MGLStyleLayer
 
-// Paint properties
+#pragma mark - Accessing the Paint Attributes
 
+/**
+  Circle radius.
+*/
 @property (nonatomic) CGFloat circleRadius;
 
+/**
+  The color of the circle.
+*/
 @property (nonatomic) MGLColor *circleColor;
 
+/**
+  Amount to blur the circle. 1 blurs the circle such that only the centerpoint is full opacity.
+*/
 @property (nonatomic) CGFloat circleBlur;
 
+/**
+  The opacity at which the circle will be drawn.
+*/
 @property (nonatomic) CGFloat circleOpacity;
 
+/**
+  The geometry's offset. Values are [x, y] where negatives indicate left and up, respectively.
+*/
 @property (nonatomic) NSArray *circleTranslate;
 
+/**
+  Control whether the translation is relative to the map (north) or viewport (screen)
+*/
 @property (nonatomic) MGLCircleStyleLayerCircleTranslateAnchor circleTranslateAnchor;
 
+/**
+  Controls the scaling behavior of the circle when the map is pitched. The value `map` scales circles according to their apparent distance to the camera. The value `viewport` results in no pitch-related scaling.
```

Comment:
We’re going to need a way to map `map` to `MGLCircleStyleLayerCirclePitchScaleMap` and so on.